src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area {
+
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public Figure() {
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public abstract String draw();
```

Comment:
[Don't put all behavior into a single interface if the methods are conceptually different from each other.](https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#dont-put-all-behavior-into-a-single-interface-if-the-methods-are-conceptually-different-from-each-other)

Create interface

Reply:
Not fixed, create Drawable interface with method void draw(); and implements AreaCalculator, Drawable
Remove public abstract String draw();